db/migrate/20160114011126_create_indices.rb

```diff
@@ -0,0 +1,13 @@
+class CreateIndices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :indices do |t|
+      t.datetime :finalized_at
+      t.integer :category_id
```

Comment:
There's still the open question of  'category'  or 'awrvi_version' (with the monkey dances required to hook it up to Category. 

The first is easy. The second better describes intent.  I *think* I prefer the second (awrvi_version), but I'm not sure of all the ramifications on setting that up and what it might do to us down the road.